docs/quiz-toolkit/gmail.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: "Gmail"
+sidebar_position: 3
+---
+# Gmail Integration
+
+Users can automatically send personalized emails to quiz participants or colleagues, directly from MecSimCalc by calling [`send_gmail`](../mecsimcalc-library#send_gmail). 
+
+In this part, we will teach you how to find the `app_password` parameter for `send_gmail` function
+### **How to Create a Google App Password**
```

Comment:
Need more info. We need to walk them through creating this app step by step. Include code e.t.c